src/main/java/org/gradle/profiler/mutations/support/ProjectCombinationsSupport.java

```diff
@@ -0,0 +1,44 @@
+package org.gradle.profiler.mutations.support;
+
+import com.google.common.collect.Sets;
+import com.google.common.math.IntMath;
+
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static org.gradle.profiler.mutations.ApplyProjectDependencyChangeMutatorConfigurator.APPLIED_PROJECTS_COUNT_KEY;
+
+public class ProjectCombinationsSupport {
+
+    /**
+     * Just to prevent some collisions with user's projects
+     */
+    public static final String PROJECT_HASH = "4b038a";
```

Comment:
I think instead of something like this we should use a prefix like `generated-dependency-`. If you have a subproject called `generated-depednency-2` yourself, then... well, you can submit a PR and then we can make the prefix configurable. Let's not make this too defensive.

Reply:
removed hash